### ISSUE TITLE
Request additional information for paper replication

### ISSUE THREAD

**Issue by TaeniKim** [PeterWang512/FALdetector#32]
I'd like to proceed with this paper as a term project.
Please upload additional data and code used to write the paper.

1) test data sets used in the paper
  (unmanipulated 50ea + maniputated 50ea)
  => Comparison of results after evaluation of the same data usage as the paper
  
2) Photoshop-enabled scripts used to create a manipulated dataset
  => Generate additional images using scripts other than the validation data you provided

3) 2AFC calculation logic(code) among the global classification evaluation items in the paper

4) EPE, IOU-3 calculation logic(code) among the local evaluation items in the paper

Please give me a positive reply.
Thank you..!!

**Comment by PeterWang512** [comment]
1. https://drive.google.com/file/d/1mzNxCyrUTBF7-lQGPLYT0HuUODvVvtsb/view
   Training set and validation set will not be released due to licensing issue
2. https://github.com/PeterWang512/FALdetector/issues/3
3. For each paired real and fake image, compare the model output of the two images. If the model's output indicates the fake image is more fake than the real one, then this serves as a correct count for 2AFC. You can then take (total correct count) / (total pairs) to get 2AFC
4. EPE: for each pixel, take the Euclidean distance between the GT and predicted flows, and average across the pixels to get EPE for a single sample. Then, average across samples to get the metric.
    IOU-3: take the magnitude of the flow for each pixel, then generate a binary mask using 3 as the threshold. Calculate IOU between the mask version of GT and predicted flow for each sample, and average the IOUs across samples.

Hope this helps, and for more information please see the paper: https://arxiv.org/pdf/1906.05856.pdf

**Comment by TaeniKim** [comment]
Thank you for your quicky reply..^^
